package.json

```diff
@@ -83,8 +83,8 @@
   "resolutions": {
     "@types/react": "^17.0.11",
     "@types/eslint": "7.29.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react": "18.0.0",
```

Comment:
Should this have a ^ in front of it? (ditto next line)

Reply:
Definitely please. This shouldn't be a question, but rather a statement.